pages_for_recommender.txt

```diff
@@ -0,0 +1,92 @@
+anomaly_detection/bayesian_changepoints.md
```

Comment:
Is this file used anywhere? It seems redundant with the hardcoded script tags in the files themselves and a potential multiple-source-of-truth.